frameworks/Julia/Http.jl/Project.toml

```diff
@@ -0,0 +1,3 @@
+[deps]
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
```

Comment:
Maybe include a `[compat]` section here?
So can use `]up` to update manifest while respecting semver